services/distribution/src/main/java/app/coronawarn/server/services/distribution/assembly/component/BusinessRulesArchiveBuilder.java

```diff
@@ -67,15 +67,16 @@ public BusinessRulesArchiveBuilder(DigitalGreenCertificateToCborMapping dgcToCbo
       rulesArchive
           .addWritable(new FileOnDisk(exportBinaryFilename,
               dgcToCborMapping.constructCborRules(ruleType, businessRuleItemSupplier, businessRuleSupplier)));
-      logger.info(archiveName + " archive has been added to the DGC distribution folder");
+      logger.info("{} archive has been added to the DGC distribution folder", archiveName);
 
       reset();
       return Optional.of(
           new DistributionArchiveSigningDecorator(rulesArchive, cryptoProvider, distributionServiceConfig));
     } catch (DigitalCovidCertificateException e) {
-      logger.error(archiveName + " archive was not overwritten because of:", e);
+      logger.error("{} archive was not overwritten because of: {}", archiveName, e.toString());
     } catch (FetchBusinessRulesException e) {
-      logger.error(archiveName + " archive was not overwritten because business rules could not been fetched:", e);
+      logger.error("{} archive was not overwritten because business rules could not been fetched: {}", archiveName,
+          e.toString());
```

Comment:
this swallows the complete stack trace, because of the different method signature, which is used here
see:
```
void error(String format, Object... arguments)
vs.
void error(String msg, Throwable t)
```

I would highly recommend to use the second one with only one string and as second argument the throwable object!!! otherwise you'll lose valuable information in the logs, when those exceptions are raised